src/test/java/codeu/model/store/basic/ActivityStoreTest.java

```diff
@@ -0,0 +1,52 @@
+package codeu.model.store.basic;
+
+import codeu.model.data.Activity;
+import codeu.model.store.persistence.PersistentStorageAgent;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class ActivityStoreTest {
+
+  private ActivityStore activityStore;
+  private PersistentStorageAgent mockPersistentStorageAgent;
+
+  private final Activity ACTIVITY_ONE =
+      new Activity(Activity.Type.LOGGED_IN, UUID.randomUUID(), Instant.ofEpochMilli(1000), UUID.randomUUID());
+
+  @Before
+  public void setup() {
+    mockPersistentStorageAgent = Mockito.mock(PersistentStorageAgent.class);
+    activityStore = ActivityStore.getTestInstance(mockPersistentStorageAgent);
+
+    final List<Activity> activityList = new ArrayList<>();
+    activityList.add(ACTIVITY_ONE);
+    activityStore.setActivities(activityList);
+  }
+
+  @Test
+  public void testAddActivity() {
+    Activity inputActivity =
+        new Activity(Activity.Type.NEW_CONVERSATION, UUID.randomUUID(), Instant.now(), UUID.randomUUID());
+
+    activityStore.addActivity(inputActivity);
+    List<Activity> activityList = activityStore.getAllActivities();
+
+    Activity resultActivity = activityList.get(1);
+
+    assertEquals(inputActivity, resultActivity);
+    Mockito.verify(mockPersistentStorageAgent).writeThrough(inputActivity);
+  }
+
+  private void assertEquals(Activity expectedActivity, Activity actualActivity) {
```

Comment:
assertEquals is a little too generic, in the test it might not be clear if you are using a custom function or something off a library. Rename to something like `assertEquivalentActivities`